components/gitpod-cli/pkg/gitpod/server.go

```diff
@@ -50,10 +59,15 @@ func ConnectToServer(ctx context.Context, wsInfo *supervisor.WorkspaceInfoRespon
 	if err != nil {
 		return nil, xerrors.Errorf("failed getting token from supervisor: %w", err)
 	}
+
 	client, err := serverapi.ConnectToServer(wsInfo.GitpodApi.Endpoint, serverapi.ConnectToServerOpts{
 		Token:   clientToken.Token,
 		Context: ctx,
 		Log:     log.NewEntry(log.StandardLogger()),
+		ExtraHeaders: map[string]string{
+			"User-Agent":       "gitpod/cli",
+			"X-Client-Version": Version,
```

Comment:
how did you verify that it is provided? 

@geropl There is not way to see it in traces?